### DISCUSSION TITLE
Sequential playlist

### DISCUSSION THREAD

**Discussion by porsasys** [AzuraCast/AzuraCast#7266] [General]
Hello,
Can anyone help me, I created a sequential playlist, I renamed the names of each song sequentially (1,2,3...), but it doesn't work, I've done everything, sometimes just one song plays and that's it. It goes to another playlist, sometimes it plays in descending sequential order (4,3,2,1), but anyway, I can't make it work.
I created the list, added the agenda and renamed the songs in sequential order (1.mp3, 2.mp3....), should I do something else to make this work?
Thank you for your help

![Captura de tela 2024-06-30 090242](https://github.com/AzuraCast/AzuraCast/assets/173726938/6700b4b4-9011-4564-b305-690039642364)
![Captura de tela 2024-06-30 090234](https://github.com/AzuraCast/AzuraCast/assets/173726938/02360ab8-41f7-45db-855c-cf9e0780983f)
![Captura de tela 2024-06-30 090222](https://github.com/AzuraCast/AzuraCast/assets/173726938/0856775b-456d-4415-b82f-a23f136c7d6f)
![Captura de tela 2024-06-30 090203](https://github.com/AzuraCast/AzuraCast/assets/173726938/17dd4204-2787-42fe-8261-2e8066222d9a)

**Comment by gAlleb** [comment]
What about 
<img width="500" alt="image" src="https://github.com/AzuraCast/AzuraCast/assets/144447266/286f98b2-6321-464a-8a93-c5dab6fb2eb5">

You don't have to rename your tracks. You ought to set order via options.

Then `sequential playlist` will start at time given. If your playlist starts at 22:30 but you restart broadcasting or docker container at 22:35 - then you could expect only one song from the playlist to be played - that is something to keep in mind.

Apart from it - everything should work, your settings look good. Just reorder your playlist via options.

If you want to test things out - be sure that you set `starting time` of your playlist some minutes after current time. `sequential playlists` via Azuracast API are gentle creatures))

**Reply by porsasys** [reply]
Hello, thank you very much, it helped me a lot.
I also activated the merge list to play all the audios at that scheduled time, which resolved the issue of playing all the audios in this sequential list I made.
After I activated (Merge the playlist), it worked with renaming the files (mp3.1,2,3...), but I didn't know about this function that you helped me with, so I won't even need to rename the files.
Thank you very much friend, God bless.

![Captura de tela 2024-06-30 101030](https://github.com/AzuraCast/AzuraCast/assets/173726938/83d02fe5-9cea-45ce-a923-99e61e87f72c)

**Comment by t0mtaylor** [comment]
Would be great to have an orderby option in the playlist so it will reorder automatically by date (new or old) or file size, so there is no user intervention when you update playlists or playlist from folders

This would make a smaller file size such as a jingle play first before a larger file or track in the playlist, etc
